src/rely/Rely.rei

```diff
@@ -78,12 +83,16 @@ module MatcherTypes: {
 
 module type TestFramework = {
   module Mock: Mock.Sig;
+  include (module type of Describe);
+  include (module type of Test);
+
   let describe: Describe.describeFn(unit);
-  let describeSkip: Describe.describeFn('a);
+  let describeSkip: Describe.describeFn(unit);
```

Comment:
We could keep this one as describeFn('a), but I think that encourages patterns we don't want to encourage and we clearly won't be able to do the same for describeOnly